dmr: docker model package (#22663)
docker/docs: content/manuals/ai/model-runner.md

Model <model> removed successfully
```

### Package a model

> Packages a GGUF file into an OCI artifact, with optional licences, and pushes it to your registry.

Reviewer: ```suggestion
Packages a GGUF file into a Docker model OCI artifact, with optional licenses, and pushes it to the specified registry.
```
